simple_http_server/README.md

```diff
@@ -0,0 +1,29 @@
+Simple HTTP Server
+===============
+
+### Objective
+
+So you want to share movies with your friends and do not have a usb drive. You can use python to share these files wirelessly.
+
```

Comment:
What about rewriting the same thing with this?

New season of game of thrones is coming in July. Suppose your friend has the first episode. You badly want to see the first episode but you do not have a pen drive. What will you do ?
You can make a simple http server from command line which will help you take the epsiode without needing pen drive. Lets get started.